samples/matter/lock/sample.yaml

```diff
@@ -26,16 +26,9 @@ tests:
       - nrf21540dk_nrf52840
     platform_allow: >
       nrf52840dk_nrf52840 nrf21540dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf7002dk_nrf5340_cpuapp
-  sample.matter.lock.switchable_thread.nrf7002_ek:
+  sample.matter.lock.thread_wifi_switched:
```

Comment:
Since we are no longer need to program the external flash, it seems that this feature can be also, or primarily, supported on the nRF7002 DK, right?